lib/src/feature_discovery.dart

```diff
@@ -733,7 +733,7 @@ class _Content extends StatelessWidget {
   final String title;
 
   /// Can be null
```

Comment:
Can it really be null? I think that this will throw an exception.

Reply:
No it won't! Check on the `_Content` `build` method. I took care of that a few weeks ago.